_source/_posts/2020-10-09-okta-spring-security-feature-flags.md

```diff
@@ -123,15 +114,15 @@ For our purposes (and to keep things simple), you just need:
 
 {% img blog/okta-split-spring-security/spring-initializr.png alt:"Initializr" width:"600" %}
 
-Spring Initializr even makes it easy to load a pre-configured project from a [direct link](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.3.4.RELEASE&packaging=jar&jvmVersion=11&groupId=com.okta.examples&artifactId=okta-split-example&name=okta-split-example&description=Feature%20Flags%20with%20Okta%2C%20Split%20and%20Spring%20Security&packageName=com.okta.examples.okta_split_example&dependencies=web,thymeleaf,okta).
+Spring Initializr even makes it easy to load a pre-configured project from a [direct link](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.3.9.RELEASE&packaging=jar&jvmVersion=11&groupId=com.okta.examples&artifactId=okta-split-example&name=okta-split-example&description=Feature%20Flags%20with%20Okta%2C%20Split%20and%20Spring%20Security&packageName=com.okta.examples.okta_split_example&dependencies=web,thymeleaf,okta).
 
 And, you can download the project from the command line with:
 
 ```bash
 curl -G \
 --data 'type=maven-project' \
 --data 'language=java' \
---data 'bootVersion=2.3.4.RELEASE' \
+--data 'bootVersion=2.3.9.RELEASE' \
```

Comment:
If we're going to update the Spring Boot version here, we should update it in the example app on GitHub too. Then, add a changelog that links to both. For the most part, I haven't been updating versions unless it's Spring Boot 2.1 (because https://github.com/oktadeveloper/okta-blog/issues/651).

Reply:
I changed it because the direct link in the post results in this message in spring initalizr web:
The following attributes could not be handled:
Spring Boot 2.3.4.RELEASE is not available, 2.3.10 has been selected.

How should I proceed?

Reply:
Now the same happens with 2.3.9!
The following attributes could not be handled:
Spring Boot 2.3.9.RELEASE is not available, 2.3.10 has been selected.

Reply:
Do I revert to 2.3.4 then?

Reply:
Yes, I would. As long as start.spring.io changes to a 2.3.x version, I think it's OK.

Reply:
Done